deployment/ansible/roles.yml

```diff
@@ -2,13 +2,13 @@
   version: 0.1.0
 
 - src: azavea.java
-  version: 0.2.6
+  version: 0.6.1
```

Comment:
I'm getting warnings like this:
```
- azavea.opentripplanner (1.0.8) was installed successfully
 [WARNING]: - dependency azavea.git from role azavea.opentripplanner differs from already installed version
(0.1.0), skipping

 [WARNING]: - dependency azavea.java from role azavea.opentripplanner differs from already installed version
(0.6.1), skipping
```

Maybe that's not new--I didn't check what it was doing before.  It seems to be a case of the `opentripplanner` role having a unrestricted dependencies (https://github.com/azavea/ansible-opentripplanner/blob/develop/meta/main.yml#L17-L18) that presumably resolve to newer versions.

Reply:
If these also come from another role, maybe they can be left out of this file altogether. I'll see what that does.

Reply:
Can simply omit them here, so will do that. Looks like ansible role versioning support is limited: https://github.com/ansible/proposals/issues/23